common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2061,6 +2066,21 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(0)
 
+  val NETWORK_IO_CLIENT_CONFLICT_AVOID_CHOOSER_ENABLE: ConfigEntry[Boolean] =
+    buildConf("celeborn.<module>.io.conflictAvoidChooser.enable")
```

Comment:
enable => enabled

similar to the class name, `conflictAvoidChooser` is not very intuitive, though I don't have a better suggestion.